inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -173,11 +173,7 @@ target_compile_definitions(${TARGET_NAME} PUBLIC -DMKLDNN_THR=${MKLDNN_THR})
 target_link_libraries(${TARGET_NAME} PRIVATE mkldnn inference_engine inference_engine_legacy
                                              inference_engine_transformations)
 
-if(USE_CNNNETWORK_LPT)
-    target_link_libraries(${TARGET_NAME} PRIVATE inference_engine_lp_transformations_legacy)
-else()
-    target_link_libraries(${TARGET_NAME} PRIVATE inference_engine_lp_transformations)
-endif()
+target_link_libraries(${TARGET_NAME} PRIVATE inference_engine_lp_transformations)
```

Comment:
let's merge with the previous `target_link_libraries`

Reply:
done, thanks